recipe/meta.yaml

```diff
@@ -24,17 +27,20 @@ build:
   number: {{ build }}
 
   # add build string so packages can depend on
-  # mpi or nompi variants explicitly:
-  # `moab * mpi_mpich_*` for mpich
-  # `moab * mpi_*` for any mpi
-  # `moab * nompi_*` for no mpi
+  # mpi or nompi and tempest or notempest variants explicitly:
+  # `moab * mpi_mpich_tempest_*` for mpich and tempest
+  # `moab * mpi_*_tempest_*` for any mpi and tempest
+  # `moab * nompi_tempest_*` for no mpi and tempest
+  # `moab * mpi_mpich_notempest_*` for mpich and no tempest
+  # `moab * mpi_*_notempest_*` for any mpi and no tempest
+  # `moab * nompi_notempest_*` for no mpi and no tempest
```

Comment:
I don't love this syntax for selecting a specific build but I don't see a good way around it.

The equivalent in a `conda create` or `conda install` command would be:
```
conda create -n test -c conda-forge python=3.7 moab=*=mpi_mpich_tempest_*
```
as an example for creating an environment with moab, python 3.7, mpich and tempest-remap.

Reply:
Again, these should be documented somewhere in the MOAB docs once we've got them built successfully and some folks have tested to make sure they behave correctly.